aip/general/0203.md

```diff
@@ -220,6 +226,19 @@ There are some changes that *are* backwards compatible, which are as follows:
 
 ## Rationale
 
+### Identifier field behavior
+
+Resource names, the primary identifier for any compliant resource, are never
```

Comment:
Change "the primary identifier" to "the primary identifiers" to match the plural nature of "resource names" and "are never"?